generators/csharp/dynamic-snippets/src/__test__/__snapshots__/DynamicSnippetsGenerator.test.ts.snap

```diff
@@ -87,62 +99,70 @@ exports[`snippets (default) > examples > 'POST /movie (simple)' 1`] = `
 "using Acme;
 using Acme.Types;
 
-var client = new AcmeClient{
-    Token = "<YOUR_API_KEY>"
-}
-await client.Service.CreateMovieAsync(new Movie{
-    ID = "movie-c06a4ad7",
-    Prequel = "movie-cv9b914f",
-    Title = "The Boy and the Heron",
-    From = "Hayao Miyazaki",
-    Rating = 8,
-    Tag = "development",
-    Metadata = new Dictionary<string, object>(){
-        ["actors"] = new List<object>() {
-            "Christian Bale",
-            "Florence Pugh",
-            "Willem Dafoe",
-        },
-        ["releaseDate"] = "2023-12-08",
-        ["ratings"] = new Dictionary<string, object>() {
-            ["rottenTomatoes"] = 97,
-            ["imdb"] = 7.6,
+var client = new AcmeClient(
+    token: "<YOUR_API_KEY>"
+);
+
+await client.Service.CreateMovieAsync(
+    new Movie{
+        ID = "movie-c06a4ad7",
+        Prequel = "movie-cv9b914f",
+        Title = "The Boy and the Heron",
+        From = "Hayao Miyazaki",
+        Rating = 8,
+        Tag = "development",
+        Metadata = new Dictionary<string, object>(){
+            ["actors"] = new List<object>() {
+                "Christian Bale",
+                "Florence Pugh",
+                "Willem Dafoe",
+            },
+            ["releaseDate"] = "2023-12-08",
+            ["ratings"] = new Dictionary<string, object>() {
+                ["rottenTomatoes"] = 97,
+                ["imdb"] = 7.6,
+            },
         },
-    },
-    Revenue = 1000000
-}).ConfigureAwait(false)"
+        Revenue = 1000000l
+    }
+);
+"
 `;
 
 exports[`snippets (default) > exhaustive > 'GET /object/get-and-return-with-optio…' 1`] = `
 "using Acme;
 using Acme.Types.Object;
 using System.Globalization;
 
-var client = new AcmeClient{
-    Token = "<YOUR_API_KEY>"
-}
-await client.Endpoints.Object.GetAndReturnWithOptionalFieldAsync(new ObjectWithOptionalField{
-    String = "string",
-    Integer = 1,
-    Long = 1000000,
-    Double = 1.1,
-    Bool = true,
-    Datetime = DateTime.Parse("2024-01-15T09:30:00Z", null, DateTimeStyles.AdjustToUniversal),
-    Date = DateOnly.Parse("2023-01-15"),
-    UUID = "d5e9c84f-c2b2-4bf4-b4b0-7ffd7a9ffc32",
-    Base64 = "SGVsbG8gd29ybGQh",
-    List = new List<string>(){
-        "list",
-        "list",
-    },
-    Set = new List<string>(){
-        "set",
-    },
-    Map = new Dictionary<int, string>(){
-        [1] = "map",
-    },
-    Bigint = "1000000"
-}).ConfigureAwait(false)"
+var client = new AcmeClient(
+    token: "<YOUR_API_KEY>"
+);
+
+await client.Endpoints.Object.GetAndReturnWithOptionalFieldAsync(
+    new ObjectWithOptionalField{
+        String = "string",
+        Integer = 1,
+        Long = 1000000l,
+        Double = 1.1,
+        Bool = true,
+        Datetime = DateTime.Parse("2024-01-15T09:30:00Z", null, DateTimeStyles.AdjustToUniversal),
+        Date = DateOnly.Parse("2023-01-15"),
+        UUID = "d5e9c84f-c2b2-4bf4-b4b0-7ffd7a9ffc32",
```

Comment:
This is probably a casing setting, but the default should be `Uuid`

Reply:
Yea that will be the case - this is just using an IR that was produced with the special casing convention (but this actually won't be used in practice for C#).